packages/component-library/components/Spacing/util.ts

```diff
@@ -14,7 +14,7 @@ export const convertFractionToString = (fraction: GridFractions): string => {
     case 1:
       return "1"
     case 1.25:
-      return "0-point-25"
+      return "1-point-25"
```

Comment:
This appears to be the root of the issue and might have been bust since the original component was created

Reply:
Is there a risk people have used this thinking it was 1.25 and are happy with the result?

Reply:
I'd say it's unlikely. If they were after 0.25, they would've gone with 0.25. If anything, I expect if they saw the 1.25 didn't look right, they'd just not use the token.

Reply:
Yeah, feel like peeps would have avoided it or used classnameOverride to fix it if they really needed it 🤷 technically still a fix so here's hoping people haven't done something strange 😬